app/templates/submit.hbs

```diff
@@ -156,6 +166,18 @@
                                     </div>
                                 </div>
                                 <div class="row">
+                                    <div class="col-md-6">
+                                        <h4>{{if newNode 'Apply' 'Also apply'}} this license to my OSF Project.</h4>
```

Comment:
Also, the spacing seems a little off here.
![screen shot 2016-12-01 at 5 57 36 am](https://cloud.githubusercontent.com/assets/9755598/20791466/2c86bd8c-b78b-11e6-8c94-cd7a456add67.png)